crates/sui-core/src/authority_active.rs

```diff
@@ -190,11 +190,17 @@ where
     A: AuthorityAPI + Send + Sync + 'static + Clone,
 {
     pub async fn spawn_all_active_processes(self) {
-        self.spawn_active_processes(true, true).await
+        self.spawn_active_processes(true, true, CheckpointProcessControl::default())
+            .await
     }
 
     /// Spawn all active tasks.
-    pub async fn spawn_active_processes(self, gossip: bool, checkpoint: bool) {
+    pub async fn spawn_active_processes(
+        self,
+        gossip: bool,
+        checkpoint: bool,
+        checkpoint_process_control: CheckpointProcessControl,
```

Comment:
This allows us to input a custom `CheckpointProcessControl` with smaller timeouts than the defaults (useful for integration tests).

Reply:
fixing in a separate PR (this one auto-merged)